source/includes/_orders.md

```diff
@@ -693,3 +693,116 @@ CANCELLED | Delivery is cancelled
 DONE | Delivery task is done
 UNKNOWN | Delivery is unknown
 QUEUED | Delivery is on queue
+
+
+## List all Orders
+
+> Definition
+
+```
+GET https://api.shareactor.io/orders/search
+```
+
+> Example request:
+
+``` http
+GET /orders/search?query=51Q4LN HTTP/1.1
+Content-Type: application/json
+Authorization: Bearer <jwt>
+X-Share-Api-Key: <shareactor-api-key>
+Host: api.shareactor.io
+```
+
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+[
+    {
+       "units":2,
+       "delivery_status":"ACCEPTED",
+       "currency":"NOK",
+       "human_id":"51Q4LN",
+       "payments":[],
+       "modified":{"$date":1495196003948},
+       "provider":{...},
+       "top_up_amount":0.0,
+       "stripe_charge_id":"",
+       "order_status":"CREATED",
+       "company":{"$oid":"591ee15db70e2a10acb65362"},
+       "billing_address":{
+          "city":"Danielsen",
+          "service":"google",
+          "alias":"",
+          "country":"Paraguay",
+          "zip_code":"0556",
+          "state":"Oslo",
+          "street_name":"Iversenstien 7"
+       },
+       "deliveries":[],
+       "items":[
+          {
+             "quantity":60,
+             "discount":0.9,
+             "product":{
+                "properties":{},
+                "parents":[],
+                "path":"/",
+                "default_position":[-1, -1],
+                "created":{"$date":1494479087000},
+                "_id":{"$oid":"591ee15eb70e2a10acb65374"},
+                "tags":[],
+                "max_distance":0,
+                "main_product":true,
+                "modified":{"$date":1495195998969},
+                "deleted":false,
+                "name":"Ivar",
+                "vat":0.0,
+                "company":{"$oid":"591ee15db70e2a10acb65362"},
+                "active":true,
+                "_cls":"Product",
+                "currency":"USD",
+                "company_take":-1.0,
+                "_sub_products":[],
+                "business_rules":[],
+                "price":100.0
+             }
+          }
+       ],
+       "note":"",
+       "created":{
+          "$date":1495264401233
+       },
+       "delivery_address":{
+          "city":"Danielsen",
+          "service":"google",
+          "alias":"",
+          "country":"Paraguay",
+          "zip_code":"0556",
+          "state":"Oslo",
+          "street_name":"Iversenstien 7"
+       },
+       "total_quantity":120,
+       "_id":{"$oid":"591ee163b70e2a10acb653a7"},
+       "total_amount":1200.0,
+       "stripe_refund_id":"",
+       "override_company_take":-1.0,
+       "user":{...},
+       "delivery_time":{"$date":1497009600000}
+    }
+]
+```
+
+Retrieves a list of all Orders associated with the search.
```

Comment:
Please put this under the header. And put in the fields that you could search for.